object-manipulation-3/main.js

```diff
@@ -0,0 +1,126 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var cardSuit = ['clubs', 'diamonds', 'hearts', 'spades'];
+var additionalCard = ['Jack', 'Queen', 'King', 'Ace'];
+
+var cardsInfo = {
+  cardsDeck: [],
+  cardUsed: []
+};
+
+var players = [{
+  name: 'player 1',
+  hand: [],
+  point: 0
+}, {
+  name: 'player 2',
+  hand: [],
+  point: 0
+}, {
+  name: 'player 3',
+  hand: [],
+  point: 0
+}, {
+  name: 'player 4',
+  hand: [],
+  point: 0
+}
+];
+var temp = {};
+for (var i = 0; i < cardSuit.length; i++) {
+  for (var j = 2; j <= 10; j++) {
+    temp.rank = j;
+    temp.suit = cardSuit[i];
+    temp.cardValue = j;
+    cardsInfo.cardsDeck.push(temp);
+    temp = {};
+    if (j === 10) {
+      for (var k = 0; k < additionalCard.length; k++) {
+        temp.rank = additionalCard[k];
+        temp.suit = cardSuit[i];
+        if (additionalCard[k] === 'Ace') {
+          temp.cardValue = 11;
+        } else {
+          temp.cardValue = 10;
+        }
+        cardsInfo.cardsDeck.push(temp);
+        temp = {};
+      }
+    }
+  }
+}
```

Comment:
This is a bit needlessly complex. Instead, you should make an array of the `suits` and an array of the `ranks` for the cards, and loop over both of them to generate a full deck.